de.metas.salescandidate.base/src/main/java/de/metas/ordercandidate/api/OLCandRepository.java

```diff
@@ -141,6 +145,8 @@ private I_C_OLCand createAndSaveOLCandRecord(@NonNull final OLCandCreateRequest
 		olCandPO.setPresetDateInvoiced(TimeUtil.asTimestamp(request.getPresetDateInvoiced()));
 		olCandPO.setC_DocTypeInvoice_ID(DocTypeId.toRepoId(request.getDocTypeInvoiceId()));
 
+		olCandPO.setC_DocTypeOrder_ID(DocTypeId.toRepoId(request.getDocTypeOrderId()));
```

Comment:
make sure we don't run into an exception but go with the normal sales order if the JSON-request contained no order-DocType

Reply:
done